RPi3-installation.md

```diff
@@ -0,0 +1,28 @@
+# Installing LibreElec, KODI, and Netflix on a Raspberry Pi
+
+These instructions assume you have an unused Raspberry Pi and are starting from scratch. The steps will wipe whatever is on the Raspberry Pi's SD card.
+
+This has been tested on a Raspberry Pi 3, but the steps are likely to be very similar if not the same for other models.
+
+## Installation the latest Alpha LibreElec and KODI 18
+
+* Download the [LibreElec USB-SD Creator](https://libreelec.tv/downloads/)
+* Install the latest LibreElec stable version onto your Raspberry Pi's SD card using the USB-SD Creator 
+* Insert the SD card into the Raspberry Pi and boot it
+* Enable SSH and Samba (SMB) options when asked - make note of the default user and password
+* Download the [latest LibreElec Alpha](https://forum.kodi.tv/showthread.php?tid=298461) (These instructions were tested using 0722, but the latest build should generally work)
+* Connect to the Updates folder on the Raspberry Pi over SMB
+* Copy the LibreElec Alpha .tar file to the Updates folder on the Raspberry Pi
+* Reboot the Raspberry Pi - it should detect and install the alpha version
+* Enable SSH and Samba (SMB) options when asked again
```

Comment:
Maybe we should change it to say enable them again "if asked" or "if prompted." Once Kodi 18 is released the instructions will need to change a bit anyway, since you won't need to install v17 to begin with.